seahorn/jobs2/array_list_capacity2/aws_array_list_capacity_harness2.c

```diff
@@ -20,16 +20,15 @@ int main() {
     assume(list.item_size > 0);
 
     /* save current state of the data structure */
-    struct aws_array_list old = list;
-    struct store_byte_from_buffer old_byte;
-    save_byte_from_array((uint8_t *)list.data, list.current_size, &old_byte);
+    sea_tracking_on();
 
     /* perform operation under verification */
     size_t capacity = aws_array_list_capacity(&list);
 
     /* assertions */
     sassert(aws_array_list_is_valid(&list));
-    assert_array_list_equivalence(&list, &old, &old_byte);
+    sassert(!sea_is_modified((char *)&list));
```

Comment:
Here we are not checking that the region pointed to by `alloc` is unmodified. Rather we are checking that the `ptr` is not changed. This check is as strong as before but weaker than the one in `aws_array_list_clear_harness2.c`. We have to choose one.

Reply:
I didn't check `alloc` on any harnesses that didn't originally check `alloc` in their implementation in `jobs`. 
![image](https://user-images.githubusercontent.com/48863772/140457704-1e4ea573-42ff-4cb9-8d19-b87297c3b1cc.png)
Do you want me to add in the check that `alloc` is unmodified if the operation under verification should not touch it?

Reply:
I have added `alloc` checks to all `array_list_equivalence` replacements and `allocator` checks to all `byte_buf_equivalent` replacements.